src/scripts/main.js

```diff
@@ -1 +1,37 @@
-'use strict';
+const track = document.querySelector('.carousel__track');
+const slides = Array.from(track.children);
+const nextButton = document.querySelector('.carousel__button-right');
+const prevButton = document.querySelector('.carousel__button-left');
+
+const slideWidth = slides[0].getBoundingClientRect().width;
```

Comment:
if I'm not mistaken, the only place where you're using this *slideWidth* variable is inside *setSlidePosition* function.
If this is the case, move this variable declaration inside the function to prevent some errors.
Like what is happening in this image here:
![image](https://github.com/mate-academy/layout_dia/assets/54222238/d31ed6ed-21d8-4ce0-8ffd-f6922b0389c5)